talent/application-process.md

```diff
@@ -52,22 +65,24 @@ The interviews consist of the following:
 
-- In the case of rejection, the developer should be notified in person on the day by the lead of the corresponding team. This should be a constructive discussion giving the developer as much insight as possible. 
+- In the case of rejection, the candidate will be notified in the following days after the interview. This will be a constructive discussion giving the candidate as much insight as possible. 
 
 ### Offer
 
-- The offer is sent by our CTO through Lever.
+- The offer is sent by our CTO or Talent Acquisition Manager once a decision to hire has been made by the team
+
+- The candidate has no concrete time deadline to accept/reject the offer, and we are happy to have another call to discuss any questions or doubts the candidate may have. In cases where we have a limited number of open positions for a role, and a larger number of people we'd like to make an offer to, a time limit is given.
 
-- The developer has no time deadline to accept/reject the offer. In cases where we have a limited number of open positions for a role, and a larger number of people we'd like to make an offer to, a time limit is given.
+### Diversity and Remote Working
 
-### Diversity and Remote
+- At source{d} we believe that amazing talent should not be constrained by physical barriers, and we exhaust all avenues to hire the best candidates regardless of their location. Our multinational and multicultural team shares a culture of openness and excellence and passion for programming.
 
-- At source{d} we believe that amazing talent should not be constrained by physical barriers. Our multinational and multicultural team shares a culture of openness and excellence and passion for programming.
+- At the moment, we have more than 35 people from 10 different countries working closely together from our office in Madrid (Spain) and across Europe & the USA. We are more than happy to help with Visa sponsorship and guide you and your family through the whole process if you decide to come to work from our Madrid office. Currently, we have remote team members in USA, Portugal, Ireland, France, Belgium, Poland, Estonia and Russia.
 
-- At the moment, we have more than 25 people from 10 different countries working closely together from our office in Madrid (Spain) and across Europe & the USA. We are more than happy to help with Visa sponsorship and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely. Currently, we have remote team members in USA, Portugal, Ireland, France, Belgium, Poland, Estonia and Russia.
+- The local timezone of candidates who want to work remotely should be between San Francisco and Moscow.
 
-- The local timezone of developers who want to work remotely should be between San Francisco and Moscow.
+- In specific cases, such as sales roles covering a particular region, there may be a requirement 
```

Comment:
I think this paragraph may be missing something.

Reply:
yeah I got to the end and just gave up, it should read _there may be a requirement to be based from a specific location_